### PR TITLE
Feat/net097 46/implement a unit commander profile

### PR DESCRIPTION
**Pull request by 5haki6**

### CODE REVIEW

File: MilitaryManager.Units.API/appsettings.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=BAGUTA;Initial Catalog=Units;User ID=sa;Password=password1;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
+    "DefaultConnection": "Data Source=localhost,1433;Initial Catalog=Units;User ID=sa;Password=password1;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
```
Comment (Alex-knu) [inline MilitaryManager.Units.API/appsettings.json]: remove

File: MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  styleUrls: ['./profile.component.scss']
+})
+
+export class ProfileComponent implements OnInit {
+
+  userName: string | null | undefined = '';
+  weight: number = 0;
+  height: number = 0;
+  rank: string | null | undefined = '';
+  position: string | null | undefined = '';
```
Comment (Alex-knu) [inline MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts:17]: ?

File: MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+              this.height = this.unitModel?.profiles["6"].value
+              // @ts-ignore
+              this.weight = this.unitModel?.profiles["5"].value
+              this.rank = this.unitModel?.rank
+              this.position = this.unitModel?.position
+              // @ts-ignore
+              this.footSize = this.unitModel?.profiles["0"].value
+              // @ts-ignore
+              this.headSize = this.unitModel?.profiles["1"].value
+              // @ts-ignore
```
Comment (Alex-knu) [inline MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts]: ?

File: MilitaryManager/client-app/src/app/layout/app.menu.component.ts [hunk 4]
```diff
@@ -20,6 +20,12 @@ export class AppMenuComponent implements OnInit {
                     { label: 'Головна', icon: 'pi pi-fw pi-home', routerLink: ['/home'] }
                 ]
             },
+            {
+               label: 'Профіль',
```
Comment (Alex-knu) [inline MilitaryManager/client-app/src/app/layout/app.menu.component.ts]: conect to icon on top

**Review by Alex-knu — CHANGES_REQUESTED** [review]

File: MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+              this.unitModel = result;
+              this.userName = this.unitModel?.lastName?.concat(" " + <string>this.unitModel?.firstName + " ").concat(<string>this.unitModel?.secondName);
+              // @ts-ignore
+              this.height = this.unitModel?.profiles["6"].value
+              // @ts-ignore
+              this.weight = this.unitModel?.profiles["5"].value
+              this.rank = this.unitModel?.rank
+              this.position = this.unitModel?.position
+              // @ts-ignore
+              this.footSize = this.unitModel?.profiles["0"].value
```
Comment (Alex-knu) [inline MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts]: rewrite like firstOrDefault(item => item.Name == "PropertyName")

**Review by Alex-knu — CHANGES_REQUESTED** [review]

**Review by Zhykovskiy — APPROVED** [review]

File: MilitaryManager.Core/DTO/Units/UnitDTO.cs [hunk 6]
```diff
@@ -21,7 +22,8 @@ public class UnitDTO
         public string Rank { get; set; }
         public UnitDTO Parent { get; set; }
         public string Position { get; set; }
-        public List<ProfileDTO> Profiles { get; set; }
+        //public List<ProfileDTO> Profiles { get; set; }
```
Comment (Alex-knu) [inline MilitaryManager.Core/DTO/Units/UnitDTO.cs]: Remove comments

File: MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts [hunk 7]
```diff
@@ -0,0 +1,41 @@
+import { Component, OnInit } from '@angular/core';
+import {UnitUserService} from "../../../shared/services/api/unit-user.service";
+import {AuthService} from "../../../shared/services/auth.service";
+import {UnitsService} from "../../../shared/services/api/unit.service";
+import {UnitModel} from "../../../shared/models/unit.model";
+
+
```
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/components/profile/profile/profile.component.ts]: Remove empty lines.

**Review by Alex-knu — APPROVED** [review]

**Pull request merged**